wynnset/tapestry-wp
# 768/841 - Show errors when submitting a node with invalid video or if there are backend errors

Since errors can occur in both the WP API and other places while submitting new nodes, we wish to provide better information about these errors to the user doing the submission. Furthermore, one example of a non-API error is the video URL problem described in #841.  This PR does the following:

- [x] displays a message to the user in the node submission box telling them an error has occurred

- [x] handles other errors related to video node submission that are currently not caught.

- [x] tests this new behaviour using cypress e2e tests

Some things to test:
1. See the new cypress tests and replicate the steps there if needed
2. Try opening the node modal, then logging out in a different window, then clicking submit. An error should appear in the modal.
